configs/resnet/resnet50_8xb32-coslr-preciseBN_in1k.py

```diff
@@ -0,0 +1,10 @@
+_base_ = 'resnet50_8xb32-coslr_in1k.py'
+
+# precise BN hook
+custom_hooks = [
+    dict(
+        type='PreciseBNHook',
+        num_items=8192,
+        interval=1,
+        priority='ABOVE_NORMAL')
```

Comment:
May add comments here to indicate why its priority is ABOVE_NORMAL.

Reply:
done.